test/Microsoft.ApplicationInsights.AspNetCore.Tests/RequestTrackingTelemetryModuleTest.cs

```diff
@@ -1,27 +1,14 @@
-﻿using Microsoft.ApplicationInsights.AspNetCore.DiagnosticListeners;
-using Microsoft.ApplicationInsights.AspNetCore.Tests.Helpers;
-using Microsoft.ApplicationInsights.Channel;
-using Microsoft.ApplicationInsights.DataContracts;
-using Microsoft.ApplicationInsights.Extensibility;
-using Microsoft.ApplicationInsights.Extensibility.Implementation;
-using Xunit;
-
-namespace Microsoft.ApplicationInsights.AspNetCore.Tests
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Tests
 {
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Xunit;
+
     public class RequestTrackingTelemetryModuleTest
     {
         [Fact]
         public void RequestTrackingTelemetryModuleDoesNoThrowWhenAppIdProviderisNull()
         {
             RequestTrackingTelemetryModule requestTrackingTelemetryModule = new RequestTrackingTelemetryModule(null);  
         }
-
-        [Fact]
-        public void RequestTrackingTelemetryModuleDoesNoThrowIfInitializeAfterDispose()
```

Comment:
I ended up deleting this test, because I don't understand why RequestTrackingTelemetryModule should work after it was disposed. Is there a real-life scenario and why we have such a  requirement?